sdk/src/main/java/com/kin/ecosystem/Kin.java

```diff
@@ -1,44 +1,50 @@
 package com.kin.ecosystem;
```

Comment:
the 'com' was omitted from package name in `core` and `common`, why the inconsistency? (breaking changes?)

Reply:
will change everything to start with `com.`